python/torch_mlir/backends/tosa/__init__.py

```diff
@@ -0,0 +1,70 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+from typing import Optional, Sequence, Union
+
+import torch
+import torch_mlir
+
+from ...compiler_utils import run_pipeline_with_repro_report
+
+def compile(model: torch.nn.Module,
+            example_args: Union[torch_mlir._example_arg, Sequence[torch_mlir._example_arg]],
+            use_tracing: bool = False,
+            ignore_traced_shapes = False,
+            backend_legal_ops: Optional[Sequence[str]] = None,
```

Comment:
remove backend_legal_ops argument. The whole point of this function is that it avoids the redundancy between the two.